src/main/java/frc/robot/subsystems/Intake/Intake.java

```diff
@@ -39,4 +39,12 @@ public static Intake getInstance() {
 		}
 		return instance;
 	}
+
+	public boolean hasGamepiece() {
+		return io.hasGamepiece();
+	}
+
+	public double getStateTime() {
```

Comment:
if you want to do this often you could just make `getStateTime` public in `Subsystem`